assignments/function-conversion.js

```diff
@@ -4,27 +4,52 @@
 // console.log("Function was invoked!");
 // };
 // myFunction();
+let myFunction = () => {
+  console.log("Function was invoked!");
+};
+myFunction();
 
 // let anotherFunction = function (param) {
 //   return param;
 // };
 // anotherFunction("Example");
 
+let anotherFunction = (param) => {
+  return param;
+};
+anotherFunction("Example");
+
 // let add = function (param1, param2) {
 //   return param1 + param2;
 // };
 // add(1,2);
 
+let add = (param1, param2) => {
+  return param1 + param2;
+};
+add(1,2);
+
 // let subtract = function (param1, param2) {
 //   return param1 - param2;
 // };
 // subtract(1,2);
 
+let subtract = (param1, param2) => {
```

Comment:
since you are only returning one line here you can ditch the curly braces and the word 'return'

```let subtract = (param1, param2) => param1-param2;```